components/Navbar.vue

```diff
@@ -145,173 +143,124 @@ import ExploreDropdown from '~/components/navbar/ExploreDropdown.vue'
 import ColorModeButton from '~/components/common/ColorModeButton.vue'
 import MobileLanguageOption from '~/components/navbar/MobileLanguageOption.vue'
 import { createVisible } from '@/utils/config/permision.config'
-import AuthMixin from '@/utils/mixins/authMixin'
-import ExperimentMixin from '@/utils/mixins/experimentMixin'
 import ChainSelectDropdown from '~/components/navbar/ChainSelectDropdown.vue'
 import StatsDropdown from '~/components/navbar/StatsDropdown.vue'
 import MobileNavbarProfile from '~/components/navbar/MobileNavbarProfile.vue'
 import ConnectWalletButton from '~/components/shared/ConnectWalletButton.vue'
-import { getKusamaAssetId } from '~/utils/api/bsx/query'
-import { NeoButton } from '@kodadot1/brick'
 import { ConnectWalletModalConfig } from '@/components/common/ConnectWallet/useConnectWallet'
+import { BModalConfig } from 'buefy/types/components'
+import Vue from 'vue'
 
-@Component({
-  components: {
-    NeoButton,
-    Search,
-    BasicImage,
-    MobileExpandableSection,
-    ProfileDropdown,
-    ExploreDropdown,
-    CreateDropdown,
-    ChainSelectDropdown,
-    StatsDropdown,
-    MobileNavbarProfile,
-    NavbarExploreOptions,
-    ConnectWalletButton,
-    ColorModeButton,
-    MobileLanguageOption,
-  },
-})
-export default class NavbarMenu extends mixins(
-  PrefixMixin,
-  AuthMixin,
-  ExperimentMixin
-) {
-  public showTopNavbar = true
-  public openMobileSearchBar = false
-  private fixedTitleNavAppearDistance = 85
-  private lastScrollPosition = 0
-  private artistName = ''
-  public isBurgerMenuOpened = false
-  private isMobile = window.innerWidth < 1024
+const { $store, $colorMode, $buefy, $nextTick } = useNuxtApp()
+const root = ref<Vue<Record<string, string>>>()
+const showTopNavbar = ref(true)
+const openMobileSearchBar = ref(false)
+const fixedTitleNavAppearDistance = ref(85)
+const lastScrollPosition = ref(0)
+const isBurgerMenuOpened = ref(false)
+const isMobile = ref(window.innerWidth < 1024)
+const { urlPrefix } = usePrefix()
 
-  @Ref('mobilSearchRef') readonly mobilSearchRef
+const mobilSearchRef = ref<{ focusInput: () => void } | null>(null)
 
-  get account() {
-    return this.$store.getters.getAuthAddress
-  }
+const route = useRoute()
 
-  get chain(): string {
-    return this.urlPrefix
-  }
+const account = computed(() => $store.getters.getAuthAddress)
 
-  get tokens() {
-    return ['', getKusamaAssetId(this.urlPrefix)]
-  }
+const isCreateVisible = computed(() => createVisible(urlPrefix.value))
+const isLandingPage = computed(() => route.name === 'index')
 
-  get isCreateVisible(): boolean {
-    return createVisible(this.urlPrefix)
-  }
+const isDarkMode = computed(
+  () =>
+    $colorMode.preference === 'dark' ||
+    document.documentElement.className.includes('dark-mode')
+)
 
-  get currentCollection() {
-    return this.$store.getters['history/getCurrentlyViewedCollection'] || {}
-  }
+const logoSrc = computed(() => (isDarkMode.value ? KodaBetaDark : KodaBeta))
```

Comment:
Time for logo component?

Reply:
I guess you are talking about the inline image component. I have introduced `@nuxtjs/svg` to make it in the previous PR, but finally do not merge it.
https://github.com/kodadot/nft-gallery/pull/4780/files

Reply:
```vue
<KodadotLogo>

```

Reply:
Yes, if we would like to use this kind of component, we need introduce a image loader, like `@nuxtjs/svg` https://www.npmjs.com/package/@nuxtjs/svg
<img width="807" alt="image" src="https://user-images.githubusercontent.com/31397967/220376946-11a4ab25-53b1-49f0-9d87-14d75a0ccd39.png">